apps/redi-talent-pool/src/pages/app/browse/BrowseJobseeker.tsx

```diff
@@ -91,8 +107,19 @@ export function BrowseJobseeker() {
           />
         </div>
       </div>
+      <div>
+        <Checkbox
+          name="isJobFair2022Participant"
+          checked={isJobFair2022JobListing}
```

Comment:
I set it to `{isJobFair2022JobListing || false}` otherwise I would get the warning
```
Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen.
```

Reply:
Hmm, I was trying to avoid using false as we won't have false values in the DB so I didn't want to filter wrong thing, when the checkbox is not checked. But I think there's a smarter way of doing this so I'll try that.

Reply:
I think everywhere else this is set to `undefined`, so the `false` should be only for the checkbox

Reply:
I mean `isJobFair2022JobListing` is set to `undefined`, or `true`. The `value` of the checkbox is never really read, therefore even when it is `false`, this won't be propagated.